src/Player/RendererViewContextEAGL.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  RendererViewContextEAGL.swift
+//  Smena
+//
+//  Created by Dmitrii on 07/04/2018.
+//  Copyright © 2018 Andrei Valkovskii. All rights reserved.
```

Comment:
We should change copyright here as well.

Reply:
done. changed for all files which are already in project